client/reader/feed-stream/index.jsx

```diff
@@ -140,6 +151,7 @@ var FeedStream = React.createClass( {
 
 		return (
 			<FollowingStream { ...this.props } listName={ this.state.title } emptyContent={ emptyContent } >
+				{ this.props.showBack ? <HeaderCake isCompact={ false } onClick={ this.goBack } /> : null }
```

Comment:
Makes me slightly nervous because when showBack is falsey, it'll render whatever is in showBack, which may not be null.